project_portal_parent_task/views/project_task_portal_template.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="portal_my_task" inherit_id="project.portal_my_task">
+        <xpath expr="//a[@t-field='task.project_id']/.." position="after">
+            <div class="mb8" t-if="task.sudo().parent_id">
+                <strong>Parent Task:</strong>
+                <a t-attf-href="/my/task/#{task.sudo().parent_id.id}" t-field="task.sudo().parent_id.display_name" />
```

Comment:
The controller '/my/task' is no longer used in version 16.0, it's replaced by '/my/tasks' see [here](https://github.com/odoo/odoo/blob/16.0/addons/project/controllers/portal.py#L117)